system-int-test/test/cmr/system_int_test/search/granule_spatial_search_test.clj

```diff
@@ -405,6 +405,32 @@
         [-5.95,-23.41,12.75,-23.69,11.11,-10.38,-6.62,-10.89,-5.95,-23.41]
         [whole-world polygon-with-holes-cart wide-south-cart normal-poly-cart]))
 
+    (testing "valid circle searches"
```

Comment:
Can you add tests that cross the antemeridian?

Reply:
Good point. Added tests.